File: wrapper/testscript/bsdconv.php
#!/usr/local/bin/php
<?php
$s=file_get_contents('php://stdin');
$a=bsdconv_create($argv[1]);
if($a===false){
	echo bsdconv_error()."\n";
	exit;
}
echo bsdconv($a,$s);
$i=bsdconv_info($a);
bsdconv_destroy($a);
echo "\n\n=======Conversino Info=======\n";
print_r($i);
?>

File: wrapper/testscript/bsdconv.pl
#!/usr/local/bin/perl
use bsdconv;

$cd=bsdconv::create($ARGV[0]);
if(!defined($cd)){ 
	print bsdconv::error()."\n";
	exit; 
}
while($str=<STDIN>){
print bsdconv::conv($cd, $str);
}
$i=bsdconv::info($cd);
bsdconv::destroy($cd);
print "\n=======Conversion Info=======\n";
for $k (keys %$i){
	print "$k=$i->{$k}\n";
}